cmd/openshift-install/create.go

```diff
@@ -217,3 +222,9 @@ func destroyBootstrap(ctx context.Context, directory string) (err error) {
 	logrus.Info("Destroying the bootstrap resources...")
 	return destroybootstrap.Destroy(rootOpts.dir)
 }
+
+// logComplete prints info upon completion
+func logComplete(directory string) {
+	logrus.Infof("Install complete! Run 'export KUBECONFIG=%s/auth/kubeconfig' to manage your cluster.", directory)
```

Comment:
To support folks who change directories, we might want to [`filepath.Abs(directory)`][1] and use the absolute path in the `KUBECONFIG` export.

[1]: https://golang.org/pkg/path/filepath/#Abs

Reply:
ok, will do.  I've always passed abs path to `--dir=absPathSomewhere`, so this has been working well in that way.  If that's not required then we'll need to filepath.Abs that :) testing now

Reply:
i would assume the logComplete take rootOpts.dir  and make the Abs internally

Reply:
the installer is not very windows compatible anyways right now so `%s/auth/kubeconfig` is fine for now.

Reply:
> the installer is not very windows compatible anyways right now so %s/auth/kubeconfig is fine for now.

I dunno how portable `export KUBECONFIG=...` is going to be, but at least slash direction is pretty straightforward.  I've pushed [a fixup branch][1] with ac3c3aadb31b to push the `Abs` call down into `logComplete` and use `filepath.Join`.  @sallyom, how does that look to you?

[1]: https://github.com/wking/openshift-installer/tree/installer-log-complete-fixup

Reply:
@wking @abhinavdahiya  that looks fine, I _was_ going to do it that way but was being lazy about returning errors, since logComplete originally did not return an error, and the postRun already did.

Reply:
pushed the change